### DISCUSSION TITLE
Stuck there.

### DISCUSSION THREAD

**Discussion by MichaelGrenier83** [tphakala/birdnet-go#135] [Q&A]
Hi, trying to install birdnetgo on a pie and i'm not able to start the app, it say:
`2024/04/30 18:37:57 ERROR: User 'root' is not member of audio group, add user to audio group by executing
2024/04/30 18:37:57 sudo usermod -a -G audio root`

Any idea.
Thanks

**Comment by tphakala** [comment]
Please provide more context, for example are you using docker or binary release?

**Reply by MichaelGrenier83** [reply]
So in started with a fresh os lite rasberry installation then installed docker using the apt repository installation then put in this command:
`docker run -ti -p 8080:8080 --device /dev/snd ghcr.io/tphakala/birdnet-go:latest`
And always receive this error:
`2024/04/30 18:37:57 ERROR: User 'root' is not member of audio group, add user to audio group by executing 2024/04/30 18:37:57 sudo usermod -a -G audio root`

Thanks in advance

**Reply by tphakala** [reply]
You are missing ALSA_CARD environment variable on your Docker start command, please see documentation https://github.com/tphakala/birdnet-go/blob/main/doc/installation.md#example-setup for detailed guide

**Reply by MichaelGrenier83** [reply]
Thanks everything is good now. 
Again thanks for your great work.
